internal/relic/planar/fleet/fleet.go

```diff
@@ -0,0 +1,53 @@
+package fleet
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/relic"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+)
+
+const (
+	mod = key.Modifier("fleet-of-the-ageless")
+)
+
+// Increases the wearer's Max HP by 12%. When the wearer's SPD reaches 120 or higher,
+// all allies' ATK increases by 8%.
+func init() {
+	relic.Register(key.FleetOfTheAgeless, relic.Config{
+		Effects: []relic.SetEffect{
+			{
+				MinCount: 2,
+				Stats:    info.PropMap{prop.HPPercent: 0.12},
+			},
+			{
+				MinCount: 2,
+				CreateEffect: func(engine engine.Engine, owner key.TargetID) {
```

Comment:
nit: this create is more complex, so I would move the function definition outside init like `onCheck` to help make it easier to read and reduce depth